src/main/java/org/itsallcode/openfasttrace/cli/commands/AbstractCommand.java

```diff
@@ -89,4 +94,12 @@ private void setTagFilter(final FilterSettings.Builder builder)
             builder.tags(wantedTags);
         }
     }
+
+    protected List<SpecificationItem> importItems()
+    {
+        this.oft.addInputs(this.toPaths(this.arguments.getInputs()));
```

Comment:
It would be possible to add the inputs to the ImportSettings class?

Reply:
Done.